#motor driving param
uint8 id
uint8 dir1
uint8 dir2
uint16 pwm